### PR TITLE
Fix/profile form state

### PR DESCRIPTION
**Pull request by kncaputo**
## What's this PR do?
- Adds `componentDidMount` to `ProfileForm`
- Removes conditional in state and instead sets state through `componentDidMount`
- Finish `onSubmit` function in `ProfileForm`
- Add fields for `description` and `address` to show on render of `Profile`

## Relevant tickets
- n/a

## Where should the reviewer start?
- review `ProfileForm` - specifically `onSubmit` and `componentDidMount`

## How should this be manually tested?
- Not much will be able to be tested until this is connected to the backend

## Any background context you want to provide?
- See above

## Screenshots/GIFs
- n/a

## Questions:
- n/a

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -35,8 +35,8 @@ const Profile = (props) => (
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
 
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
-        {/* TODO: <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.shippingAddress}</p> */}
-        {/* TODO: <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.description}</p> */}
+        <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.shippingAddress}</p>
```
Comment (saraho1123) [inline code/web/src/modules/user/Profile.js:38]: Excited to see this show up on our profile page!!

File: code/web/src/modules/user/ProfileForm.js [hunk 2]
```diff
@@ -28,14 +28,38 @@ class ProfileForm extends Component {
...
       isLoading: false
     }
   }
 
+  componentDidMount() {
+    let shippingAddress = '';
+    let description = '';
+    let userDetails; 
+
+    if (this.props.user.details.address) {
```
Comment (saraho1123) [inline code/web/src/modules/user/ProfileForm.js:44]: so grateful for you Kara, and for finding that the store is already in fact updated by this point!! Very much appreciate getting to work with you! :)

**Review by saraho1123 — APPROVED** [review]
Excited to get this all working with BE! Thanks y'all!

**Review by jdcarey128 — APPROVED** [review]
LGTM

**Pull request merged**